packages/vector_graphics/lib/src/vector_graphics.dart

```diff
@@ -358,12 +359,6 @@ class _VectorGraphicWidgetState extends State<VectorGraphic> {
         textDirection: key.textDirection,
         clipViewbox: key.clipViewbox,
         loader: loader,
-        onError: (Object error, StackTrace? stackTrace) {
```

Comment:
The error handler has been moved higher up, so this one should not be needed anymore.